File: launchdarkly/php/src/GetExperimentResultsForMetricGroupExample.php
<?php

namespace OSEG\LaunchDarklyExamples;

require_once __DIR__ . '/../vendor/autoload.php';

use SplFileObject;
use LaunchDarkly;

$config = LaunchDarkly\Client\Configuration::getDefaultConfiguration();
$config->setApiKey("ApiKey", "YOUR_API_KEY");

try {
    $response = (new LaunchDarkly\Client\Api\ExperimentsApi(config: $config))->getExperimentResultsForMetricGroup(
        project_key: null,
        environment_key: null,
        experiment_key: null,
        metric_group_key: null,
        iteration_id: null,
    );

    print_r($response);
} catch (LaunchDarkly\Client\ApiException $e) {
    echo "Exception when calling ExperimentsApi#getExperimentResultsForMetricGroup: {$e->getMessage()}";
}

File: launchdarkly/python/src/GetExperimentResultsForMetricGroupExample.py
import json
from datetime import date, datetime
from pprint import pprint

from launchdarkly_client import ApiClient, ApiException, Configuration, api, models

configuration = Configuration(
    api_key={"ApiKey": "YOUR_API_KEY"},
)

with ApiClient(configuration) as api_client:
    try:
        response = api.ExperimentsApi(api_client).get_experiment_results_for_metric_group(
            project_key=None,
            environment_key=None,
            experiment_key=None,
            metric_group_key=None,
            iteration_id=None,
        )

        pprint(response)
    except ApiException as e:
        print("Exception when calling ExperimentsApi#get_experiment_results_for_metric_group: %s\n" % e)
